SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceUniqueId.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+
+namespace SpatialGDK
+{
+struct EventTraceUniqueId
+{
+	uint8 internal_bytes[8] = { 0 }; // This size may need to be adjusted in the future. Usage should not be direct.
+
+	FString GetString() const;
+
+	bool IsValid() const
```

Comment:
Just wondering why this is inlined?

Reply:
Hmm no reason really, just because the implementation is simple. Could move

Reply:
Yeah don't mind much.

Reply:
I guess the calling code can completely inline the method (potentially as a single instruction) rather than invoking a function call or leaning on link-time inlining